pkg/operation/botanist/controlplane.go

```diff
@@ -474,15 +475,46 @@ func (b *Botanist) DeploySeedLogging() error {
 	}
 
 	var (
-		credentials = b.Secrets["logging-ingress-credentials"]
-		basicAuth   = utils.CreateSHA1Secret(credentials.Data[secrets.DataKeyUserName], credentials.Data[secrets.DataKeyPassword])
-		kibanaHost  = b.Seed.GetIngressFQDN("k", b.Shoot.Info.Name, b.Garden.Project.Name)
+		kibanaHost           = b.Seed.GetIngressFQDN("k", b.Shoot.Info.Name, b.Garden.Project.Name)
+		kibanaCredentials    = b.Secrets["kibana-logging-sg-credentials"]
+		sgKibanaPassword     = kibanaCredentials.Data["password"]
+		sgKibanaPasswordHash = kibanaCredentials.Data["hashedPassword"]
```

Comment:
```suggestion
		sgKibanaPasswordHash = kibanaCredentials.Data["bcryptHashedPassword"]
```

or something similar